Date: 2018-01-05
By: Bernhard M. Wiedemann <bwiedemann suse de>

PATCH-FIX-UPSTREAM in 1.174

dont add timestamps to .gz headers

generate reproducible random output for missing glyphs

Index: console-setup-1.134/Fonts/Makefile
===================================================================
--- console-setup-1.134.orig/Fonts/Makefile
+++ console-setup-1.134/Fonts/Makefile
@@ -7,7 +7,7 @@ build-any:
 
 
 %.gz : %
-	gzip -9 <$< >$@
+	gzip -n9 <$< >$@
 
 # All available non-legacy font faces
 FACES = Fixed13 Fixed14 Fixed15 Fixed16 \
Index: console-setup-1.134/Fonts/bdf2psf
===================================================================
--- console-setup-1.134.orig/Fonts/bdf2psf
+++ console-setup-1.134/Fonts/bdf2psf
@@ -21,6 +21,8 @@
 
 use warnings 'all';
 use strict;
+srand(0);
+
 
 #####################################################################
 
